frontend/app.py

```diff
@@ -1008,6 +1012,29 @@ def _(
     )
 
 
+@create_download_response.register
+def _(
+    download_request_details: DownloadWindowsFakeFSRequest, canarydrop: Canarydrop
+) -> DownloadWindowsFakeFSResponse:
+    """"""
+    fake_file_structure = windows_fake_fs.FOLDER_MAP.get(
+        canarydrop.windows_fake_fs_file_structure
+    )
+    if not fake_file_structure:
+        return JSONResponse({"message": "Something went wrong!"}, status_code=500)
```

Comment:
Should we log an error here so we know if this happens (since it shouldn't happen unless the token creation did something wrong)?

Reply:
I don't think its needed. This is there to prevent an corner case as folks pick from a list so they are not in control of sending the key for the lookup but this will catch data being manipulated